pkg/processor/build/runtime/java/docker/onbuild/Dockerfile

```diff
@@ -74,11 +77,13 @@ ONBUILD RUN mv /home/gradle/src/userHandler/build/libs/user-handler.jar /home/gr
 
 # The directory will hold wrapper source, SDK jar, user-handler.jar and wrapper gradle build script
 COPY pkg/processor/runtime/java/build.gradle \
-    pkg/processor/runtime/java/nuclio-sdk-1.0-SNAPSHOT.jar \
```

Comment:
? just `nuclio-sdk-java-1.0.0.jar \`

Reply:
I tried..it didn't work that way, the COPY command didn't returned an error when it was with that path (`COPY failed: stat /var/lib/docker/tmp/docker-builder.../nuclio-sdk-1.0-SNAPSHOT.jar :no such file or directory`
I saw it happened to others(https://github.com/docker/for-mac/issues/1922), so I used `RUN cp...` as a simple workaround here

Reply:
Weird... Ok